modules/fuse-box-loader-api/fusebox.js

```diff
@@ -276,10 +276,12 @@ function $import(name, o) {
             v: ref.versions,
         });
     };
-    locals.require.main = {
-        filename: $isBrowser ? "./" : g["require"].main.filename,
-        paths: $isBrowser ? [] : g["require"].main.paths,
-    };
+    if ($isBrowser || !g["require"].main) {
```

Comment:
`g["require"]` is only working on the server, don't you think we can revise that?